src/interfaces/announce.transactions.command.ts

```diff
@@ -15,148 +15,94 @@
-                        this.spinner.stop(true)
-                        console.log(chalk.green('\nTransaction announced.'))
-                        this.spinner.start()
-                    }
-                    else if (response instanceof Transaction){
-                        listener.close()
-                        this.spinner.stop(true)
-                        console.log(chalk.green('\nTransaction confirmed.'))
-                    }
-                }, (err) => {
-                    listener.close()
-                    this.spinner.stop(true)
-                    console.log(HttpErrorHandler.handleError(err))
-                })
-        }, (err) => {
-            listener.close()
-            this.spinner.stop(true)
-            console.log(chalk.red('Error'), err.message)
-        })
-    }
+        if (transactionAnnounceMode === TransactionAnnounceMode.normal) {return null}
 
-    /**
-     * Announces a hash lock transaction. Once this is confirmed, announces an aggregate transaction.
-     * @param {signedHashLockTransaction} Signed hash lock transaction.
-     * @param {signedAggregateTransaction} Signed aggregate transaction.
-     * @param {Address} senderAddress - Address of the account sending the transaction.
-     * @param {string} url - Node URL.
-     */
-    protected announceAggregateTransaction(signedHashLockTransaction: SignedTransaction,
-                                           signedAggregateTransaction: SignedTransaction,
-                                           senderAddress: Address,
-                                           url: string) {
-        this.spinner.start()
-        let confirmations = 0
-        const transactionHttp = new TransactionHttp(url)
-        const listener = new Listener(url)
-        listener.open().then(() => {
-            merge(
-                transactionHttp.announce(signedHashLockTransaction),
-                listener
-                    .status(senderAddress)
-                    .pipe(
-                        filter((error) => error.hash === signedHashLockTransaction.hash),
-                        tap((error) => {
-                            throw new Error(error.code)
-                        })),
-                listener
-                    .confirmed(senderAddress)
-                    .pipe(
-                        filter((transaction) => transaction.transactionInfo !== undefined
-                            && transaction.transactionInfo.hash === signedHashLockTransaction.hash),
-                        mergeMap((ignored) => transactionHttp.announceAggregateBonded(signedAggregateTransaction)),
-                    ),
+        // Get the profile's multisig accounts multisig account info
+        const profile: Profile = this.getProfile(options)
+        const childMultisigAccountsInfo = await new MultisigService(profile).getChildrenMultisigAccountInfo()
+
+        if (!childMultisigAccountsInfo) {
```

Comment:
I don't think the CLI should verify if the account belongs to a multisig account because it requires having access to the internet/node. It will be better to validate only if the CLI can communicate with the node.

In general, the user should be able to sign the transaction offline. At this moment, the only exceptions are the metadata & mosaic restrictions commands - they use a service that requires the internet - but this is something we will need to change in a future iteration.

Reply:
Closing that in favor of #275